src/jobsautoreport/query.py

```diff
@@ -65,6 +69,14 @@ def _get_query_steps_by_name(
             }
         }
 
+    def query_equinix_project_usages(self, from_date: datetime, to_date: datetime) -> list[EquinixUsage]:
+        headers = {"X-Auth-Token": self._equinix_project_token}
+        url = f"https://api.equinix.com/metal/v1/projects/{self._equinix_project_id}/usages"
+        equinix_project_usages = requests.get(url=url, headers=headers).json()['usages']
+        logger.info('usages retrieved successfully')
+        parsed_equinix_project_usages = [EquinixUsage.parse_obj(usage) for usage in equinix_project_usages if usage["end_date"] != None]
+        return [usage for usage in parsed_equinix_project_usages if usage.start_date.utcnow() >= from_date and usage.end_date.utcnow() <= to_date]
+
```

Comment:
the `Querier` provides an abstraction on top of elasticsearch. In order to split the responsibilities, we should have a dedicated class  for equinix, and inject it into the `Reporter` object.